settings.py

```diff
@@ -5,6 +5,7 @@
 
 REPOSITORY = os.getenv("INPUT_REPOSITORY")
 GH_TOKEN = os.getenv("INPUT_GH_TOKEN")
+GH_API_URL = os.getenv("GITHUB_API_URL")
```

Comment:
Add in the file `action.yml` this property not required, with a default value to not break for anyone that not define this var.

Reply:
Hi, the GITHUB_API_URL env. var is a default variable and automatically added to each action. I think it's not needed to add it. It's automatically set by the github action based on the used github.

Reply:
Forget the var import
![image](https://user-images.githubusercontent.com/13910431/159583547-d389dd70-5cbe-47aa-a6f0-e9937ebaca91.png)